### PR TITLE
Related to #465 Make events.Schedule a ConfereneRelated model

### PR DESCRIPTION
**Pull request by adrianliaw**
Note that the migration also sets the `conference` field of previous Schedules with their year of creation

Reference: #465

### CODE REVIEW

File: src/events/migrations/0014_schedule_conference.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def add_conference_to_previous_schedules(apps, schema_editor):
+    Schedule = apps.get_model('events', 'Schedule')
+    db_alias = schema_editor.connection.alias
+    for schedule in Schedule.objects.using(db_alias).all():
+        schedule.conference = 'pycontw-{}'.format(schedule.created_at.year)
```
Comment (uranusjr) [inline src/events/migrations/0014_schedule_conference.py]: smart

Comment (adrianliaw) [reply src/events/migrations/0014_schedule_conference.py]: @uranusjr 對惹其實我不知道為什麼要用 `.using(db_alias)`，是看前面的幾個 migration 有做類似的事情的時候都有這樣寫所以才這樣寫的，那個用意是什麼啊？

Comment (uranusjr) [reply src/events/migrations/0014_schedule_conference.py]: best practice in case you have multiple databases

Comment (adrianliaw) [reply src/events/migrations/0014_schedule_conference.py]: Got it! <(_ _)>

**Pull request merged**